addons/evm/src/commands/actions/deploy_contract.rs

```diff
@@ -47,97 +47,113 @@ lazy_static! {
                 typing: Type::string(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             max_fee_per_gas: {
                 documentation: "Sets the max fee per gas of an EIP1559 transaction.",
                 typing: Type::integer(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             max_priority_fee_per_gas: {
                 documentation: "Sets the max priority fee per gas of an EIP1559 transaction.",
                 typing: Type::integer(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             chain_id: {
                 documentation: "The chain id.",
                 typing: Type::string(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             nonce: {
                 documentation: "The account nonce of the signer. This value will be retrieved from the network if omitted.",
                 typing: Type::integer(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             gas_limit: {
                 documentation: "Sets the maximum amount of gas that should be used to execute this transaction.",
                 typing: Type::integer(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             gas_price: {
                 documentation: "Sets the gas price for Legacy transactions.",
                 typing: Type::integer(),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             contract: {
                 documentation: "Coming soon",
                 typing: CONTRACT_METADATA.clone(),
                 optional: false,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             constructor_args: {
                 documentation: "Coming soon",
                 typing: Type::array(Type::string()),
                 optional: true,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             confirmations: {
                 documentation: "Once the transaction is included on a block, the number of blocks to await before the transaction is considered successful and Runbook execution continues.",
                 typing: Type::integer(),
                 optional: true,
-                interpolable: true
+                tainting: false,
```

Comment:
Confirmations is tainting on the check_confirmations function, but not here. Is that right?